ci/prow/Makefile

```diff
@@ -12,14 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-CLUSTER       ?= prow
-PROJECT       ?= knative-tests
-ZONE          ?= us-central1-f
-JOB_NAMESPACE ?= test-pods
-
-PROW_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
-TESTGRID_DIR := $(PROW_DIR)/../testgrid
-SET_CONTEXT := gcloud container clusters get-credentials "$(CLUSTER)" --project="$(PROJECT)" --zone="$(ZONE)"
+CLUSTER            ?= prow
+PROJECT            ?= knative-tests
+ZONE               ?= us-central1-f
+JOB_NAMESPACE      ?= test-pods
+KNATIVE_CONFIG     ?= config_knative.yaml
+SKIP_CONFIG_BACKUP ?=
```

Comment:
what's the use case of skipping config backup? can't we pass in `PROW_CONFIG_GCS` if the concern is to avoid collision?

Reply:
I just want to have as many knobs as possible. Sample use case: a staging/temp cluster where I don't care about preserving the config.

Reply:
sgtm